orbit infra providers (#1059)
OffchainLabs/arbitrum-docs: arbitrum-docs/launch-orbit-chain/infra-options-orbit-chains.md

# Infrastructure Options for Orbit Chains

> A successful production-grade chain often requires additional surrounding infrastructure to enable app functionality and provide a suberb user experience for those interacting with the network. There are multiple options for various components for deployers to choose from, and the below is a non-comprehensive list of providers that Orbit teams can choose to work with if the need should arise.

Reviewer: ```suggestion
A successful production-grade chain often requires additional surrounding infrastructure to enable app functionality and provide a superb user experience for those interacting with the network. There are multiple options for various components for deployers to choose from, and the below is a non-comprehensive list of providers that Orbit teams can choose to work with if the need should arise.
```

> A successful production-grade chain often requires additional surrounding infrastructure to enable app functionality and provide a suberb user experience for those interacting with the network. There are multiple options for various components for deployers to choose from, and the below is a non-comprehensive list of providers that Orbit teams can choose to work with if the need should arise.

Reviewer 2: ```suggestion
A successful production-grade chain often requires additional infrastructure to enable app functionality and provide a good user experience. Orbit teams can choose from multiple options for various components. The following list is not comprehensive.
```

## Chain Operation

> Many application developers do not have the subject matter expertise required to deploy, operate, and maintain an independent production-grade network. In these cases, we highly recommend working with one of the following RaaS providers who will manage the necessary infrastructure and ensure a high degree of performance and security for your chain:

Reviewer 2: ```suggestion
Many application developers do not have the subject matter expertise required to deploy, operate, and maintain an independent, production-grade network. 
In these cases, we highly recommend working with one of the following RaaS (Rollup as a Service) providers who will manage the necessary infrastructure and ensure a high degree of performance and security for your chain:
```

- [Caldera](https://www.caldera.xyz/)
- [Conduit](https://conduit.xyz/)
- [AltLayer](https://altlayer.io/)
- [Gelato](https://www.gelato.network/)

## Explorers

> Explorers are a basic requirement for any functional blockchain, providing a transparent view of transactions, blocks, and addresses on the blockchain. They can be accessed by users and developers to verify transactions, check the status of network activity, and ensure that transactions are being processed correctly. Some providers that can offer an explorer to Orbit developers are:

Reviewer 2: ```suggestion
Explorers are essential for any functional blockchain, providing a transparent view of transactions, blocks, and addresses on the blockchain. Users and developers can access them to verify transactions, check the status of network activity, and ensure that transactions are being processed correctly. Some providers that can offer an explorer to Orbit developers are:
```

- [Blockscout](https://www.blockscout.com/)
- [Socialscan](https://socialscan.io/)
- [Lore](https://www.lorescan.com/)
- [Routescan](https://routescan.io/)

> Additionally, Orbit chains leveraging blobs for data availability may use tools like [Blobscan](https://blobscan.com/) to see which blob/block included a given transaction.

Reviewer 2: ```suggestion
Additionally, Orbit chains leveraging blobs for data availability may use tools like [Blobscan](https://blobscan.com/) to see which blob/block includes a given transaction.
```

## Bridges

> Orbit chains are easily launched with a canonical token bridge which permits transfers to and from their chain via Arbitrum One, Nova, or Ethereum. However, some applications may desire the ability to transfer to additional chains outside of the Orbit ecosystem or want to offer the ability to transfer across chain more quickly, without waiting for complete finality. In these instances, the following 3rd party bridging providers offer services to meet those needs:

Reviewer 2: ```suggestion
Orbit chains are easily launched with a canonical token bridge, which allows transfers to and from their chain via <a data-quicklook-from="arbitrum-one">Arbitrum One</a>, <a data-quicklook-from="arbitrum-nova">Nova</a>, or Ethereum. However, some applications may require the ability to transfer to chains outside the Orbit ecosystem or across chains faster without waiting for complete finality. In these instances, the following 3rd party bridging providers offer services meeting those needs:
```

- [LayerZero](https://layerzero.network/)
- [Connext](https://www.connext.network/)
- [Hyperlane](https://www.hyperlane.xyz/)
- [Axelar](https://axelar.network/)
- [Across](https://across.to/)

## Data Availability

> Deployers of Orbit chains may offer cheaper fees on their chain by storing the data posted in batches on an alternative Data Availability solution rather than to Ethereum directly. The built in functionality in Anytrust allows these developers to easily configure the cost of transacting and will enable more powerful features with additional chains that are also using Anytrust. However, there are integrations with Orbit that are available for those that would prefer to use one of the following third party services:

Reviewer: ```suggestion
Deployers of Orbit chains may offer cheaper fees on their chain by storing the data posted in batches on an alternative Data Availability solution rather than to the parent chain directly. The built in functionality in Anytrust allows these developers to easily configure the cost of transacting and will enable more powerful features with additional chains that are also using Anytrust. However, there are integrations with Orbit that are available for those that would prefer to use one of the following third party services:
```
Not sure if you intended to include Ethereum sxplicitly here, but pointing it out just in case.

> Deployers of Orbit chains may offer cheaper fees on their chain by storing the data posted in batches on an alternative Data Availability solution rather than to Ethereum directly. The built in functionality in Anytrust allows these developers to easily configure the cost of transacting and will enable more powerful features with additional chains that are also using Anytrust. However, there are integrations with Orbit that are available for those that would prefer to use one of the following third party services:

Reviewer 2: ```suggestion
As an Orbit chains deployer, you may offer cheaper fees on your chain by storing the data posted in batches on an alternative Data Availability solution rather than directly storing it in the parent chain. AnyTrust's built-in functionality allows you to set transaction fees easily and will enable more powerful features with additional chains that also use AnyTrust. Alternatively, you can choose from the following third-party services:
```

- [Celestia](https://celestia.org/)
- [EigenDA](https://www.eigenlayer.xyz/)
- [Avail](https://www.availproject.org/) (coming soon!)
- [Near](https://near.org/data-availability) (coming soon)

## Indexers

> In order to surface necessary application information, teams frequently require additional indexing capabilities. Fetching application specific or historical information is difficult or impossible to do directly from an RPC. Fortunately, there are multiple data services providers that are familiar with operating this type of infrastructure, and may be willing to do so for your chain.

Reviewer 2: ```suggestion
Teams frequently require additional indexing capabilities to retrieve historic or application-specific data, but doing so directly from an RPC endpoint is difficult or impossible. Fortunately, multiple data services providers are familiar with operating this type of infrastructure and may be willing to do so for your chain.
```

- [Alchemy](https://www.alchemy.com/)
- [The Graph](https://thegraph.com/)
- [Goldsky](https://goldsky.com/)
- [Ormi](https://www.ormilabs.xyz/)

## Oracles

> Many applications require external data to be published on-chain in order to function properly. This data is usually either price feeds related to assets utilized in app or a source of verifiable randomness. The following oracle providers may be able to support your chain:

Reviewer 2: ```suggestion
Many applications require external data to be published on-chain to function properly. This data usually consists of price feeds related to assets utilized in-app or a source of verifiable randomness. The following Oracle providers may be able to support your chain:
```
